test/server/test_data/server/bad_sds_config_source.yaml

```diff
@@ -4,7 +4,6 @@ node:
 static_resources:
   clusters:
   - name: xds-grpc
-    connect_timeout: 0.25s
```

Comment:
not sure if this is part of this files badness - but i think it would just be ignore when the yaml was parsed